templates/desktop/organisations.html

```diff
@@ -560,194 +565,44 @@ <h3>Landscape</h3>
     </div>
     <div class="col-6">
       <h3>Active Directory</h3>
-      <p>Integrating Ubuntu Desktop into an existing Active Directory architecture can be an automated and effortless process when using System Security Services Daemon (SSSD). GPO is available by default to manage desktop settings and Ubuntu Advantage users receive access to additional Active Directory policies like remote script execution and privilege management.</p>
-      <p><a href="/engage/microsoft-active-directory">Integrating Ubuntu Desktop with Active Directory&nbsp;&rsaquo;</a></p>
-      <p><a href="https://github.com/ubuntu/adsys/wiki">ADSys documentation&nbsp;&rsaquo;</a></p>
+      <p>Integrating Ubuntu Desktop into an existing Active Directory architecture can be an automated and effortless process when using System Security Services Daemon (SSSD). GPO is available by default to manage desktop settings and Ubuntu Pro users receive access to additional Active Directory policies like remote script execution and privilege management.</p>
+      <p><a href="/engage/microsoft-active-directory" class="p-button--positive">Integrating Ubuntu Desktop with Active Directory</a></p>
+      <p><a href="https://github.com/ubuntu/adsys/wiki" class="p-button">ADSys documentation</a></p>
```

Comment:
```suggestion
      <p><a href="https://github.com/ubuntu/adsys/wiki" class="p-button">ADSys documentation</a></p>
      <p><a href="/engage/microsoft-active-directory" class="p-button--positive">Integrating Ubuntu Desktop with Active Directory</a></p>
```
Should swap these around as in the doc. In general the 'positive' cta's will be the final item